subsys/bluetooth/mesh/Kconfig

```diff
@@ -107,13 +107,21 @@ config BT_MESH_PB_GATT
 	  GATT.
 
 config BT_MESH_GATT_PROXY
-	bool "GATT Proxy Service"
+	bool "GATT Proxy Service support"
```

Comment:
GATT Proxy Service feature?

Reply:
"support" is used for the rest of the mesh features, I'd prefer to stay consistent